CHANGELOG.md

```diff
@@ -21,7 +21,13 @@ The new repository can be included in projects as follows. As of version 7.0.0,
         <url>https://pkgs.dev.azure.com/sovity/5bec6cbd-c80a-47ac-86ce-1deb26cee853/_packaging/artifact/maven/v1</url>
```

Comment:
Is there a possibility to get a more handy URL?

Reply:
At least I can't find a way in Azure after a short look. Azure DevOps -> Messaging-Services -> Artifacts -> Connect to feed -> Maven:

![image](https://user-images.githubusercontent.com/75306992/170244007-10fd1a7e-6169-4943-9e72-ee4cf98d8641.png)

Reply:
I will resolve this, since I cant see a way. If there's a way, we can still change it in the future.